packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -4300,14 +4300,14 @@ describe('shallow', () => {
     it('works with a name', () => {
       const wrapper = shallow(<div />);
       wrapper.single('foo', (node) => {
-        expect(node).to.equal(wrapper.node);
+        expect(node).to.equal(wrapper[sym('__node__')]);
```

Comment:
Can these underscores be added by the `sym` function, instead of hardcoded in the text?